pom.xml

```diff
@@ -55,14 +40,12 @@
 	</properties>
 
 	<dependencies>
-
 		<dependency>
-			<groupId>org.codehaus.jackson</groupId>
-			<artifactId>jackson-mapper-asl</artifactId>
-			<version>1.6.4</version>
+			<groupId>com.fasterxml.jackson.core</groupId>
+			<artifactId>jackson-databind</artifactId>
+			<version>2.13.4</version>
```

Comment:
zou je deze versie niet in het properties gedeelte zetten met een variable of word dit in dit project helemaal niet gedaan?

Reply:
Niet consequent, maar ik heb deze in de properties gezet